.github/workflows/deploy-canary.yml

```diff
@@ -74,10 +74,15 @@ jobs:
       - uses: actions/checkout@v3
         with:
           token: ${{secrets.MERGE_ACTION}}
+<<<<<<< HEAD
       - name: Use Node.js 14.x
         uses: actions/setup-node@v3
+=======
+      - name: Use Node.js 16.x
+        uses: actions/setup-node@v2
+>>>>>>> a9064d28cd (chore(deps): upgrade to node 16)
```

Comment:
Be careful with this :)

(there is more like this in the PR)

Reply:
Huh, thanks for pointing this out! Not sure what's up, my editor didn't show any conflicts.